bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -85,4 +92,14 @@ default boolean getAutopauseDisabled() {
     default String getAccessibilityLabel() {
         return null;
     }
+
+    /**
+     * Checks if the control elements should be placed in front of the carousel items.
+     *
+     * @return {@code true} if the control elements should be placed in front of the items, {@code false} if they should be appended
+     * @since com.adobe.cq.wcm.core.components.models 12.20.0
+     */
+    default boolean getPrependControls() {
```

Comment:
I'd rename it isControlsPrepended()

Reply:
adressed in a407f8573ac5905bd5d9573fdc05fca0a2753c36